frontend/src/store/slices/accessRequests/accessRequestsSlice.ts

```diff
@@ -0,0 +1,90 @@
+import { IAccessRequestsFilterData, IAccessRequestsState } from './interfaces';
+import { IAccessRequest, IPagedItems } from 'interfaces';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { ACCESS_REQUEST } from 'constants/reducerTypes';
+
+export const MAX_ACCESS_RESULTS_PER_PAGE = 100;
+// First, define the reducer and action creators via `createSlice`
+
+/**
+ * convert an unstructured object into a properly formatted access request.
+ * @param values
+ */
+export const toAccessRequest = (values: any): IAccessRequest => {
+  return {
+    id: values.id,
+    userId: values.userId,
+    user: {
+      id: values.userId,
+      username: values.user.username,
+      email: values.user.email,
+      position: values.user.position,
+    },
+    agencies: isNaN(values.agency) ? [] : [{ id: parseInt(values.agency) }],
+    roles: [{ id: values.role }],
+    status: values.status,
+    note: values.note,
+    rowVersion: values.rowVersion,
+  };
+};
+
+export const initialState: IAccessRequestsState = {
+  pagedAccessRequests: { page: 1, pageIndex: 0, total: 0, quantity: 0, items: [] },
+  filter: { agency: '', role: '', searchText: '' },
+  sorting: { column: 'username', direction: 'desc' },
+  selections: [],
+  accessRequest: null,
+  pageSize: MAX_ACCESS_RESULTS_PER_PAGE,
+  pageIndex: 0,
+};
+/**
+ * The access request slice manages the current access request, a list of all active access requests, and methods to sort and filter this list.
+ */
+export const accessRequestsSlice = createSlice({
+  name: ACCESS_REQUEST,
```

Comment:
Why not just put the string value here?  What value is there in having a separate file full of strings that shouldn't ever be referenced by anything else anyways?